README.md

```diff
@@ -225,55 +156,29 @@ Example:
 
 > Describe 1 - 3 features you will add to the project.
 
-- [ ] **[new_feature_1]**
-- [ ] **[new_feature_2]**
-- [ ] **[new_feature_3]**
+no new features will be added
```

Comment:
- [ ] That's an honest entry there 👍 but how about we pretending that we are going to add some more styling, html content, add nice animations, and dynamism to the site? 😄 This shows that we are conscious of growth and continuous improvement.